### ISSUE TITLE
Crash when using hasOwnProperty

### ISSUE THREAD

**Issue by binary64** [shuckster/eslint-plugin-big-number-rules#2]
My code:

```
Object.prototype.hasOwnProperty.call({})

export default null
```
crashes with
```
$ C:\p\monorepo\node_modules\.bin\eslint apps/web/components/MyComponent.tsx

Oops! Something went wrong! :(

ESLint: 7.32.0

TypeError: Cannot convert undefined or null to object
Occurred while linting C:\p\monorepo\apps\web\components\MyComponent.tsx:1
    at hasOwnProperty (<anonymous>)
    at C:\p\monorepo\node_modules\eslint-plugin-big-number-rules\dist\index.js:7:3903
    at C:\p\monorepo\node_modules\eslint\lib\linter\safe-emitter.js:45:58
    at Array.forEach (<anonymous>)
    at Object.emit (C:\p\monorepo\node_modules\eslint\lib\linter\safe-emitter.js:45:38)
```

**Comment by shuckster** [comment]
Thanks for the report, looking into it. 👀

**Comment by shuckster** [comment]
@binary64 Should be fixed in `1.7.7`. Please let me know if it works for you.
